client_grpc.go

```diff
@@ -156,7 +156,7 @@ func headerInterceptor(headers map[string]string) connect.UnaryInterceptorFunc {
 	}
 }
 
-func (c *clientGrpc) getToken() (*getTokenResult, error) {
+func (c *clientGrpc) GetToken() (*getTokenResult, error) {
```

Comment:
this returns a private type from a public interface, which is allowable but bad practice

you can instead return a new interface type that implements an interface that `*getTokenResult` implements. Further, to use this without knowing the type (which is also private -- `clientGrpc`), this method should be lifted up to `Client` defined in `chalk-go/client.go:9`, as otherwise this type will be inaccessible

Reply:
will do!